ui/src/app/applications/components/utils.tsx

```diff
@@ -40,6 +43,20 @@ export function helpTip(text: string) {
 }
 export async function deleteApplication(appName: string, apis: ContextApis): Promise<boolean> {
     let confirmed = false;
+    const propagationPolicies: {name: string; message: string}[] = [
+        {
+            name: 'Foreground',
+            message: 'Delete the cascading resources in the foreground followed by the application'
+        },
+        {
+            name: 'Background',
+            message: 'Delete the application followed by the cascading resources in the background'
+        },
+        {
+            name: 'Orphan',
+            message: 'Delete the application leaving the resources orphaned'
+        }
```

Comment:
I think we should ensure some more consistency within the `message` field, which is displayed as tool-tip to the user. Also, the switch from "Cascade" to "Orphan" in terminology may be a little confusing, especially because we use the term "Orphan" (unfortunately) elsewhere too with a different semantic.

My proposal would be:

```

        {
            name: 'Foreground',
            message: 'Cascade delete the applications resources using foreground propagation policy'
        },
        {
            name: 'Background',
            message: 'Cascade delete the applications resources using background propagation policy'
        },
        {
            name: 'Non-cascading',
            message: 'Only delete the application, but do not cascade delete its resources'
        }
```

WDYT?

Reply:
I like this.

Reply:
Is it `application's`?

Reply:
You may be right there, @jsoref - but as a non-native speaker, I don't have a final answer to that :) I guess we could also say `the resources of the application` instead of `the application's resources`

Reply:
`-s`=plural, `'s`=possessive, so it sounds like you want the latter.

Reply:
(`its` is an exception to this, where `its` is possessive and `it's` is a contraction for `it is`)

Reply:
Will `application's resources` cause confusion since we're not setting the propagation policy for the resources themselves? ie. If we delete the application with background policy, do the individual K8s resources get deleted with the same background propagation policy?

Reply:
Sorry for the confusion! I was under the assumption that the propagation policy is for the `Application` and not for the Application's resources. Thanks to @jannfis for clearing my doubts. All good now, I've updated the controller in the PR to update the policy for the Application's resources.